tests/vcs/git/GitCommitBearTest.py

```diff
@@ -288,6 +288,24 @@ def test_body_checks(self):
                              body_regex=r'TICKER\s*CLOSE\s+[1-9][0-9]*'), [])
         self.assert_no_msgs()
 
+        # Testing invalid emails
+        self.git_commit('Shortlog\n\n'
+                        'invalid#!format!*!@email.com\n'
+                        'another.*invalid@format\n'
+                        'validformat@email.check\n'
+                        'validdomain@gmail.com\n'
+                        'Fix 2017')
+        message = 'Body contains these invalid emails:\n'
+        message += ' invalid#!format!*!@email.com\n'
+        message += ' another.*invalid@format\n'
+        # Test with email_dns_check set to False
+        self.assertEqual(self.run_uut(), [message])
+        # Test with email_dns_check set to True
+        message += ' validformat@email.check\n'
+        self.assertEqual(self.run_uut(email_dns_check=True), [message])
```

Comment:
Is the test here actually doing the DNS lookup? If so, please mock it away, we don't want out test suite to be network dependent.